resources/assets/js/components/Forms/TextArea.tsx

```diff
@@ -12,6 +12,10 @@ export interface TextAreaProps {
   errorText?: string;
   onChange: (event: React.ChangeEvent<HTMLTextAreaElement>) => void;
   onBlur?: (event: React.ChangeEvent<HTMLTextAreaElement>) => void;
+
+  // formik
+  field?: any;
```

Comment:
@yonikid15 's PR #1052 will affect this, but we decided to keep Formik-specific components in the `Form/` directory and leave the existing input components in the `components/` directory.

Reply:
Yes, I want to get this code in. I will happily help refactor for Yoni's upcoming change.

Reply:
Yoni is working on this.